DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py

```diff
@@ -625,25 +630,29 @@
     # Select events based on the pixel cluster multiplicity
     import  HLTrigger.special.hltPixelActivityFilter_cfi
     process.multFilter = HLTrigger.special.hltPixelActivityFilter_cfi.hltPixelActivityFilter.clone(
-        inputTag  = 'siPixelClusters',
+        inputTag = 'siPixelClusters',
         minClusters = 1,
         maxClusters = 50000
-        )
```

Comment:
No idea (let me know if you think any further changes are needed).

Reply:
>  No idea (let me know if you think any further changes are needed).

I think it would be useful if Tracker DPG (@cms-sw/trk-dpg-l2) could take a look.

Reply:
Hi, @sroychow and I discussed a little bit about this selection (I added it in 2018...). From what I was able to reconstruct it was added to avoid processing too large events which could crash the online client. Now, if after the changes on the HLT selection this is not needed anymore we can remove it but I personally don't know the impact of the changes done at HLT level.

Reply:
>  Now, if after the changes on the HLT selection this is not needed anymore

what was the reason to include `HLT_HICentralityVeto*` (presumably selecting pp-like events) and mix it with        `'HLT_HIPhysics*` presumably selecting dense events?